doc/nrf/libraries/zigbee/osif.rst

```diff
@@ -114,6 +114,9 @@ Use the following options to configure the Zigbee serial logger:
   * :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME` - This option specifies the serial device to use.
   * :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_BUFFER_SIZE` - This option specifies the size of the internal ring buffer.
 
+  .. note::
+    See :ref:`zephyr:usb_device_cdc_acm` for more information about how to configure USB CDC ACM instance for logging ZBOSS trace messages.
```

Comment:
Why not add this line or note next to line 113?
Also, please fix formatting of indent for lines 111-115 -- `*` should start from the beginning of the line.